.licignore

```diff
@@ -5,9 +5,9 @@
 !Makefile
 !*.go
 !Dockerfile*
+!hack/boilerplate.gotxt
```

Comment:
it's totally in your hands, just, isn't enough to limit range of txt files in `.licignore` , so the other txt will not be affected and than add rule for `.txt` in `.golic.yaml` ? 
```
!hack/boilerplate.go.txt
```